_includes/community/community-meetings-table.html

```diff
@@ -0,0 +1,64 @@
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<div align="center"><h3 class="black-text">Layer5 Meetings</h3></div>
+
+<div style="overflow-x:auto;">
+<table id="table101">
+<tr>
+  <th>Group</th>
+  <th>Slack Channel</th>
+  <th>Meeting Minutes</th> 
+  <th>Meeting Link</th>
+  <th>Meeting Recordings</th>
+</tr>
+<tr>
+  <td><b><font color = #000000>Layer5 Community Meeting</font></b></td>
```

Comment:
Avoid inline style when you can. Use quotation marks around color. `<b>` and `<font>` are deprecated tags. Use `class=black-text`.

Reply:
Remove emphasis (use normal font weight) for all values in this column.